README.md

```diff
@@ -17,6 +17,8 @@ In this application, you can record live or upload an audio file form your compu
 # Contributing as a Developer
 
 ## Development Environment
+
+* If you are on MacOS, please refer to the macos-setup.md file.
```

Comment:
Could include all operating systems on the new markdown file rather than only focusing it on MacOS

Reply:
I agree as I have been having issues installing the requirements on Windows.

Reply:
Consider expanding the macos-setup.md to include all OS setups or adding separate sections for Windows. This will make it easier for all developers to follow.